### ISSUE TITLE
BulkItemChange not working for tagged values

### ISSUE THREAD

**Issue by chrislro2019** [Helmut-Ortmann/EnterpriseArchitect_hoTools#6]
I like to set tagged value(s) on selected component(s). Therefor I modified settings.json as below for bulk change. It does set the Version to 2.0 as expected, but it does not set the tagged value in EA 15.2.1555 and hoTools 3.8.0. 

```
"BulkItems": [
    {
      "Name": "Set Component Tag foo to bar",
      "Description": "",
      "TypesCheck": ["Component"],
      "StereotypesCheck": [""],
      "StereotypesApply": [],
      "TaggedValuesApply": [
        {
          "Name": "foo",
          "Value": "bar"
        }
      ],
      "PropertiesApply": ["Version=2.0"]
    }
]
```

1. I recognized in the code of BulkChangeElement() that it does require that the tagged value is already present at the element. I would have expected, that it **adds** a non-existent tagged value. https://github.com/Helmut-Ortmann/EnterpriseArchitect_hoTools/blob/f76b3617215699bdc142c1f594be0d457e048fbc/Util/BulkChange/BulkItemChange.cs#L130-L141
2. So I tried to add the tagged value first manually for testing, but also the bulk change does not work, the tagged value is not changed

So am I doing something wrong?

**Comment by Helmut-Ortmann** [comment]
Hello,

I'll look into this and will come back.

At first glance, it looks good. I think I can reproduce it.

In principle, I often use this functionality but sometimes there is ......

Best regards,

Helmut

**Comment by Helmut-Ortmann** [comment]
I've reproduced it and will fix it in next release.

**Comment by Helmut-Ortmann** [comment]
Fixed in 3.8.2
